content/docs/concepts/observability/index.md

```diff
@@ -0,0 +1,159 @@
+The operations guide on [Envoy Statistics](/docs/ops/telemetry/envoy-stats/) provides more information on controlling the generation of proxy-level metrics.
+
+Example proxy-level Metrics:
+
+```json
+envoy_cluster_internal_upstream_rq{response_code_class="2xx",cluster_name="xds-grpc"} 7163
+
+envoy_cluster_upstream_rq_completed{cluster_name="xds-grpc"} 7164
+
+envoy_cluster_ssl_connection_error{cluster_name="xds-grpc"} 0
+
+envoy_cluster_lb_subsets_removed{cluster_name="xds-grpc"} 0
+
+envoy_cluster_internal_upstream_rq{response_code="503",cluster_name="xds-grpc"} 1
+```
+
+### Service-level Metrics
+
+In addition to the proxy-level metrics, Istio provides a set of service-oriented metrics for monitoring service communications. These metrics cover the four
+basic service monitoring needs: latency, traffic, errors, and saturation. Istio ships with a default set of
+[dashboards](/docs/tasks/telemetry/metrics/using-istio-dashboard/) for monitoring service behaviors based on these metrics.
+
+The [default Istio metrics](/docs/reference/config/policy-and-telemetry/metrics/) are defined by a set of configuration artifacts that ship with Istio and are
+exported to [Prometheus](http://localhost:1313/docs/reference/config/policy-and-telemetry/adapters/prometheus/) by default. Operators are free to modify the
+shape and content of these metrics, as well as to change their collection mechanism, to meet their individual monitoring needs.
+
+The [Collecting Metrics](/docs/tasks/telemetry/metrics/collecting-metrics/) task provides more information on customizing Istio metrics generation.
+
+Use of the service-level metrics is entirely optional. Operators may choose to turn off generation and collection of these metrics to meet their individual
+needs.
+
+Example service-level metric:
+
+```json
+istio_requests_total{
+  connection_security_policy="mutual_tls",
+  destination_app="details",
+  destination_principal="cluster.local/ns/default/sa/default",
+  destination_service="details.default.svc.cluster.local",
+  destination_service_name="details",
+  destination_service_namespace="default",
+  destination_version="v1",
+  destination_workload="details-v1",
+  destination_workload_namespace="default",
+  reporter="destination",
+  request_protocol="http",
+  response_code="200",
+  response_flags="-",
+  source_app="productpage",
+  source_principal="cluster.local/ns/default/sa/default",
+  source_version="v1",
+  source_workload="productpage-v1",
+  source_workload_namespace="default"
+} 214
+```
+
+### Control Plane Metrics
+
+Each Istio component (Pilot, Galley, Mixer) also provides a collection of self-monitoring metrics. These metrics allow monitoring of the behavior
+of Istio itself (as distinct from that of the services within the mesh).
+
+For more information on which metrics are maintained, please refer to the reference documentation for each of the components:
+
+- [Pilot](/docs/concepts/traffic-management/overview/#pilot)
```

Comment:
This doesn't seem right. The sentence says refer to the "reference documentation", but this link points to concepts doc.